### PR TITLE
MapStruct + Spring boot data JPA

### PR DESCRIPTION
**Pull request by OjAntoni**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,6 +28,13 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct</artifactId>
+            <version>1.4.2.Final</version>
```
Comment (wrlk) [inline pom.xml]: Best practice is to keep deps versions as pom properteis

File: pom.xml [hunk 2]
```diff
@@ -37,9 +44,38 @@
...
     <build>
         <plugins>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.6.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
```
Comment (wrlk) [inline pom.xml]: Why 1.8 here? It seems you use at least java 11

File: src/test/java/by/innowise/monolithwebcalculator/domain/mapper/OperationMapperTest.java [hunk 3]
```diff
@@ -8,21 +8,21 @@
...
-
+import org.mapstruct.factory.Mappers;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 
 class OperationMapperTest {
 
-    static OperationMapper mapper;
+    static OperationMapper operationMapper;
```
Comment (wrlk) [inline src/test/java/by/innowise/monolithwebcalculator/domain/mapper/OperationMapperTest.java]: You can use `@TestInstance` annotation to make `@BeforeAll` method be non static. Do not declare static class members instead.

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1 +1,4 @@
-
+spring.datasource.url=jdbc:postgresql://localhost:5432/innowise
```
Comment (OjAntoni) [inline src/main/resources/application.properties]: I have a problem with connecting to postgres container from outside (It tells me to check if the port accept tcp/ip)

File: src/main/java/by/innowise/monolithwebcalculator/domain/mapper/OperationMapper.java [hunk 5]
```diff
@@ -3,18 +3,22 @@
...
-        return op;
-    }
-
-    public OperationResponseDto mapOperationToOperationResponseDto(Operation op) {
-        return new OperationResponseDto(op.getArgOne(), op.getArgTwo(), op.getResult(), op.getOperationType());
-    }
+@Mapper(componentModel = "spring")
+public interface OperationMapper {
+    @Mappings({
+            @Mapping(target = "argOne", source = "dto.argOne"),
```
Comment (AlexLink90) [inline src/main/java/by/innowise/monolithwebcalculator/domain/mapper/OperationMapper.java]: Is it necessary to write "dto." before argOne and argTwo? I think mapper will understand without it

**Pull request merged**